src/test/java/com/wikia/webdriver/testcases/mediatests/VetAddingVideoTests.java

```diff
@@ -25,25 +25,25 @@
 
   Credentials credentials = Configuration.getCredentials();
 
-  @Test(groups = {"VetAddVideo_001"})
+  @Test(groups = {"VetAddVideo_001"}, invocationCount = 20)
   @RelatedIssue(issueID = "QAART-889", comment = "test fails randomly")
   @Execute(asUser = User.USER)
   public void VetAddVideo_001_MessageWallProvider() {
     MessageWall wall = new MessageWall(driver).open(credentials.userName);
     String message = PageContent.MESSAGE_WALL_MESSAGE_PREFIX + wall.getTimeStamp();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "message" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1481)